deploy/olm-catalog/kubefed-operator/0.1.0/kubefed-operator.v0.1.0.clusterserviceversion.yaml

```diff
@@ -304,75 +304,124 @@ spec:
+    The upstream user guide uses `kube-federation-system` as the name of this namespace.
 
-    Before you use kubefed, you should ensure that you're using the binary
-    for this version. You can get it from the releases on GitHub.
+    ### Download binaries
 
-        curl -Ls https://github.com/kubernetes-sigs/kubefed/releases/download/v0.1.0-rc3/kubefedctl-0.1.0-rc3-linux-amd64.tgz | tar xz
+    Once the `kubefed` control plane is up and running, a user can download the `kubefedctl` command-line tool to communicate with it. 
+    You can get the latest release on [GitHub](https://github.com/kubernetes-sigs/kubefed/releases).
 
+    ```
+     $ curl -Ls https://github.com/kubernetes-sigs/kubefed/releases/download/v0.1.0-rc4/kubefedctl-0.1.0-rc4-linux-amd64.tgz
+     $ tar -xvzf kubefedctl-0.1.0-rc4-linux-amd64.tgz
+     $ chmod u+x kubefedctl
+     $ sudo mv kubefedctl /usr/local/bin/ # make sure the location is in the PATH
+    ```
     ### Joining Clusters
 
-    Use the `kubefedctl join` command to connect clusters you want federation to
-    spread resources to. `kubefedctl join` reads information about how to connect
-    to the joining cluster and the cluster hosting federation from your
-    KUBECONFIG.
-
-        kubefedctl join cluster-name \
-          --cluster-context mycluster \            # name of a KUBECONFIG context for the cluster to join
-          --host-cluster-context mycluster \       # name of a KUBECONFIG context for the hosting cluster
-          --kubefed-namespace=<namespace> \        # namespace where federation is deployed
-          -v 2
-     
-     Note: if you're using an openshift cluster for deploying the kubefed-operator then
+    Use the `kubefedctl join` command to connect clusters you want federation to spread resources to. The `join` operation reads information 
+    about how to connect to the joining cluster and the cluster hosting federation from your KUBECONFIG.
+
+    ```
+    $ kubefedctl join cluster-name \
+        --cluster-context mycluster \            # name of a KUBECONFIG context for the cluster to join
+        --host-cluster-context mycluster \       # name of a KUBECONFIG context for the hosting cluster
+        --kubefed-namespace=<namespace> \        # namespace where federation is deployed
+        -v 2
+    ``` 
+    **Note**: If you're using an openshift cluster for deploying the kubefed-operator then
      use the following command for renaming context to consumable format.
+     
+     ```
+     $ kubectl config rename-context $(kubectl config current-context) <cluster-name>
+     
+     ```
 
-     oc config rename-context $(oc config current-context) <cluster-name>
-
-    `kubefedctl join` creates `KubeFedCluster` and `Cluster` resources named
-    `cluster-name` to represent the joined cluster.
+     kubefedctl join creates `KubeFedCluster` and `Cluster` resources named `cluster-name` to represent the joined cluster.
```

Comment:
Shouldn't `kubefedctl join` continue to be enclosed in ticks since it is a command?

Also `Cluster` resources are no longer relevant since KubeFed no longer uses the cluster registry.  `join` only creates a KubeFedCluster resource and associated secret.